scripts/cluster_providers/kind_get_cluster_auth

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+EXISTING_CLUSTER=KIND_$(echo $CLUSTER_VERSION | sed -e "s/\./\_/g")
```

Comment:
@pedjak First we detect if there is any existing cluster present. 
`EXISTING_CLUSTER=KIND_$(echo $CLUSTER_VERSION | sed -e "s/\./\_/g")`

If we are creating cluster using `kind` in the script then kubectl context would be set using 
`echo "kubectl config set-context $1`

Reply:
even if the cluster exists, it will be created previously using `kind`, because we are here at `kind` provider.  And even if it exists, the only important thing here is the cluster name. In both cases you invoke here the same command `kubectl config set-context $1`. `ClusterProvider.py` knows the cluster name and passes it to the script.